apps/trustlab/src/payload/collections/Pages.js

```diff
@@ -29,6 +26,14 @@ const Pages = {
 
       return `${process.env.NEXT_PUBLIC_APP_URL}/preview?${encodedParams.toString()}`;
     },
+    components: {
+      edit: {
+        PublishButton:
+          "@/trustlab/payload/components/PagesPublishButton#PagesPublishButton",
+        PreviewButton:
+          "@/trustlab/payload/components/PagesPreviewButton#PagesPreviewButton",
+      },
+    },
```

Comment:
Do we need this? If we've already set access to edit pages to editor/admins only, I assume authors won't be able to see pages altogether or they'll see them in readonly mode.